### DISCUSSION TITLE
thank you!

### DISCUSSION THREAD

**Discussion by t0mZ4** [syagent/agent-2#2] [General]
having used nodequery for the last 5 years just to have it drop of the face of the planet one day

dug around and for your tool 

so far looks very impressive, will use it an provide feedback/suggestions as we go!

thanks

**Comment by Rashe** [comment]
Hey, thanks. It would be very helpful to understand what syagent miss as a feature

**Comment by t0mZ4** [comment]
NP

Is there an alert if the server is offline?

**Reply by Rashe** [reply]
Of course.

It's called "agent stopped receiving data". 

In short how it works:
When you add new server to monitor, syagent service waits for data from your server. If syagent service didn't receive the data 6 times (6 mins), it will raise an alert. 

Btw, you can customize the alerts you want to receive in server edit.

**Comment by t0mZ4** [comment]
thanks! so yea i didn't see this specific alert there is why I asked:

![image](https://user-images.githubusercontent.com/31729091/144481516-6647bbc6-0ffc-48c7-859d-26ee8baf9843.png)

**Reply by Rashe** [reply]
Got it now, I will add it to alarms too
